src/presign/participant.rs

```diff
@@ -305,6 +270,52 @@ impl PresignParticipant {
         Ok(vec![message])
     }
 
+    fn handle_round_two_msg<R: RngCore + CryptoRng>(
+        &mut self,
+        rng: &mut R,
+        message: &Message,
+        main_storage: &Storage,
+    ) -> Result<Vec<Message>> {
+        let (auxinfo_identifier, keyshare_identifier) =
+            self.get_associated_identifiers_for_presign(&message.id())?;
+
+        // First, verify the bytes of the round two value, and then
+        // store it locally. In order to v
```

Comment:
nit: This was here before, but please finish the sentence or remove the fragment.